packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -124,7 +124,9 @@ protected static function filter_declaration( $property, $value, $spacer = '' )
 			$filtered_value = wp_strip_all_tags( $value, true );
 
 			if ( '' !== $filtered_value ) {
-				return safecss_filter_attr( "{$property}:{$spacer}{$filtered_value}" );
+				// @TODO safecss_filter_attr can't handle background-image with multiple values.
```

Comment:
Need a core patch to test that the following `background-image` CSS is allowed by [safecss_filter_attr](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/kses.php#L2312):

- Linear gradient + image: `background-image: linear-gradient(rgba(0, 0, 255, 0.5), rgba(255, 255, 0, 0.5)), url("https://domain/some/image.png");`
- The reverse: `background-image: url("https://domain/some/image.png"), linear-gradient(rgba(0, 0, 255, 0.5), rgba(255, 255, 0, 0.5));`
- Multiple images: background-image: url("some/image.png"),  url("https://domain/some/other/image.png");
- Any other valid values, e.g., 


```css
background-image: linear-gradient(
    to bottom,
    rgb(255 255 0 / 50%),
    rgb(0 0 255 / 50%)
  ), url("catfront.png");
```

Source: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image)

Reply:
Related

- https://github.com/WordPress/gutenberg/pull/62207#issuecomment-2144232548
- https://core.trac.wordpress.org/ticket/48376
- https://core.trac.wordpress.org/ticket/24157